extensions/jib/deployment/src/main/java/io/quarkus/jib/deployment/JibConfig.java

```diff
@@ -0,0 +1,38 @@
+package io.quarkus.jib.deployment;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.BUILD_TIME)
+public class JibConfig {
+
+    /**
+     * If enabled, a container image will be created using jib
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean enabled;
+
+    /**
+     * The container registry to use
+     */
+    @ConfigItem(defaultValue = "docker.io")
+    public String registry;
+
+    /**
+     * The group the container image will be part of
+     */
```

Comment:
Missing `@ConfigItem`? Same for other properties below

Reply:
Indeed!

Reply:
Fixed